docs/data/date-pickers/localization/LocalizedTimePicker.tsx

```diff
@@ -1,22 +1,22 @@
 import * as React from 'react';
-import ruLocale from 'date-fns/locale/ru';
-import enLocale from 'date-fns/locale/en-US';
-import arSaLocale from 'date-fns/locale/ar-SA';
+import dayjs, { Dayjs } from 'dayjs';
+import ruLocale from 'dayjs/locale/ru';
+import arSaLocale from 'dayjs/locale/ar-sa';
 import Stack from '@mui/material/Stack';
 import ToggleButton from '@mui/material/ToggleButton';
 import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
 import TextField from '@mui/material/TextField';
 import MenuItem from '@mui/material/MenuItem';
 import Select, { SelectChangeEvent } from '@mui/material/Select';
-import { AdapterDateFns } from '@mui/x-date-pickers/AdapterDateFns';
+import { AdapterDayjs } from '@mui/x-date-pickers/AdapterDayjs';
 import { DateTimePicker } from '@mui/x-date-pickers/DateTimePicker';
 import { TimePicker } from '@mui/x-date-pickers/TimePicker';
 import { LocalizationProvider } from '@mui/x-date-pickers/LocalizationProvider';
 import FormControl from '@mui/material/FormControl';
 import InputLabel from '@mui/material/InputLabel';
 
 const localeMap = {
-  en: enLocale,
+  en: undefined,
```

Comment:
It's a bit surprising when seeing `undefined` for a locale
```suggestion
  // Do not need to pass `en` since it's the default one
  en: undefined,
```

Reply:
The problem is that the `en` local exported by `dayjs` is incomplete
I don't understand why.
But since it does not have all the needed formats, if you try passing it to the adapter it crashes .